ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -634,6 +634,24 @@ func (r *RayClusterReconciler) updateStatus(instance *rayiov1alpha1.RayCluster)
 		}
 	}
 
+	runtimeServices := corev1.ServiceList{}
+	filterLabels = client.MatchingLabels{common.RayClusterLabelKey: instance.Name, common.RayNodeTypeLabelKey: "head"}
+	if err := r.List(context.TODO(), &runtimeServices, client.InNamespace(instance.Namespace), filterLabels); err != nil {
+		return err
+	}
+	if len(runtimeServices.Items) != 0 {
+		svc := runtimeServices.Items[0]
```

Comment:
Let's add a comment here. Now we actually just have one service. In the future, we might have multiple and the assumption here won't work. 

If we know exact service name, we can use `Get` to replace `List` as well.